components/financial_assistance/app/controllers/financial_assistance/evidences_controller.rb

```diff
@@ -26,7 +26,7 @@ def update_evidence
     end
 
     def fdsh_hub_request
-      authorize @evidence, :fdsh_hub_request?
+      authorize HbxProfile, :can_call_hub?
```

Comment:
Just curious as to why we are refering to the Hbxprofile directly instead of what we were doing before?. All it needed was to use `can_call_hub?` instead of `can_extend_due_date?`. New changes also should be consistent with our expectation of every action should have its own policy method. Devise way of doing is for every controller should have its own policy class, and every action should have its own policy method. Though this controller is validating a different object from the one this controller is directly related to. It makes sense to have the own policy and refer the whatever policy you want from it. Any thoughts?

Reply:
@hari.yamjala Alec has responded to several requested changes by @vishal.kalletla and @harshavardhan.ellanki to implement authorization specifically in this way.  Please discuss with them if you feel this change is absolutely necessary.  Considering they are using this pattern elsewhere and prefer it, and that they were assigned review initially on it, and the priority of delivering this for QA, I am not considering this feedback a blocker to approval.  Please discuss this with them further if you feel it's necessary.